frontend/src/components/pages/CampCreation/StepperNavigation/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Box, Center } from "@chakra-ui/react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCampground,
+  faCalendarDays,
+  faCircleInfo,
+} from "@fortawesome/free-solid-svg-icons";
+import StepperTab from "./StepperTab";
+
+export enum CampCreationPages {
```

Comment:
Not that it matters all that much, but why define the pages here instead of the parent wrapper component?

Reply:
By parent wrapper component do you mean the Camp Creation index.tsx? If so, we have some circular dependencies since I'll need to import `CampCreationPages` from the camp creation index, but that index will import this Stepper index.

Reply:
ahh ok yeah that makes sense. thanks!